base_layer/wallet/src/storage/sqlite_db.rs

```diff
@@ -344,6 +485,21 @@ impl WalletBackend for WalletSqliteDatabase {
             ckv.set(&conn)?;
         }
 
+        // remove tor id encryption if present
+        let key_str = WalletSettingSql::get(format!("{}", DbKey::TorID), &conn)?;
+        match key_str {
+            Some(v) => {
+                let decrypted_key_bytes = decrypt_bytes_integral_nonce(&cipher, from_hex(v.as_str())?)
+                    .map_err(|_| WalletStorageError::AeadError("Decryption Error".to_string()))?;
```

Comment:
Swallowing an error here

Reply:
Changed the error type here to not swallow any errors anymore